web-components/src/rootStyles/design-tokens.mdx

```diff
@@ -84,7 +98,7 @@ properties.
       return numberA - numberB;
     })
     .map(([, token], index) => (
-      <React.Fragment key={index}>
+      <div key={index} style={{display: 'contents'}}>
```

Comment:
React has been removed as a peer dependency of Storbook, so since we only use it here, we can remove it and use the `display: contents` to have HTML Elements that don't interfere with the CSS Grid